Makefile

```diff
@@ -68,3 +68,11 @@ assisted_deployment:
 
 assisted_deployment_requirements:
 	./assisted_deployment.sh install_prerequisites_assisted_service
+
+fleeting: fleeting_configure requirements configure fleeting_deploy_iso bell
+
+fleeting_configure:
+	./fleeting_configure.sh
+
+fleeting_deploy_iso:
+	./fleeting_deploy_iso.sh
```

Comment:
It might be a good idea to create a subdirectory, e.g similar to how I've been doing hypershift testing:

https://github.com/hardys/dev-scripts/tree/hypershift_wip/hypershift

Then we can also have separate cleanup scripts, README, any non-shared ansible roles etc